app/models/models.rb

```diff
@@ -12,6 +12,8 @@ module QDM
 require_relative 'qdm/basetypes/interval'
 require_relative 'qdm/basetypes/quantity'
 require_relative 'qdm/basetypes/ratio'
+# FacilityLocation is technically a basetype
+require_relative 'qdm/facility_location'
```

Comment:
There is a duplicate `require_relative 'qdm/facility_location'` down below added by looping through the datatypes.   Adding `<%- next if datatype.underscore == 'facilityLocation' -%>` to the template file should fix that.